src/_lib/template.js

```diff
@@ -714,42 +852,81 @@ export async function getHtml(parsedReq) {
           instructor={instructor}
           palette={avatarPalette}
           parsedReq={parsedReq}
-        />
+        />,
       )
       break
     case 'playlists':
       const playlist = await axios
         .get(`https://egghead.io/api/v1/playlists/${parsedReq.text}`)
-        .then(({data}) => data)
-      console.log('playlist: ', playlist)
+        .then(({ data }) => data)
+
+      let itemsToMap = []
+
+      const upperLevelLessons = playlist.items.filter(
+        ({ thumb_nail }) => !!thumb_nail,
+      )
+
+      const subPlaylists = playlist.items.filter(
+        (item) => item.type === 'playlist',
+      )
+
+      if (!isEmpty(upperLevelLessons)) {
+        itemsToMap = upperLevelLessons.slice(0, 3)
+      } else if (!isEmpty(subPlaylists)) {
+        const { data } = await axios.get(subPlaylists[0].items_url)
+        itemsToMap = data.slice(0, 3)
+      }
+
+      let subPlaylistsLessonsAmount = 0
```

Comment:
I'm very unsure about the code below - I want to get all lessons from subplaylists (I mean situation when playlist is playlists of playlists, for example https://egghead.io/playlists/composing-closures-and-callbacks-in-javascript-1223)